.contentful/vault-secrets.yaml

```diff
@@ -0,0 +1,9 @@
+version: 1
+services:
+  github-action:
+    policies:
+      - dependabot
+  circleci:
+    policies:
+      - npm-read
```

Comment:
Afaik this should not be required separately since the `semantic-release-ecosystem` already has thee required NPM token